### ISSUE TITLE
unsuccessful installation

### ISSUE THREAD

**Issue by alebugariu** [moves-rwth/homebrew-storm#8]
I am trying to install Storm on a machine with macOS Ventura 13.4, Apple M1 chip. However, `brew install stormchecker` does not succeed, the `make install` step produces the following error:

```
Last 15 lines from Library/Logs/Homebrew/stormchecker/02.make:
    CMakeFiles/storm-parsers.dir/api/model_descriptions.cpp.o
    CMakeFiles/storm-parsers.dir/parser/PrismParser.cpp.o
duplicate symbol 'boost::phoenix::placeholders::uarg9' in:
    CMakeFiles/storm-parsers.dir/api/model_descriptions.cpp.o
    CMakeFiles/storm-parsers.dir/parser/PrismParser.cpp.o
duplicate symbol 'boost::phoenix::placeholders::uarg10' in:
    CMakeFiles/storm-parsers.dir/api/model_descriptions.cpp.o
    CMakeFiles/storm-parsers.dir/parser/PrismParser.cpp.o
ld: 90 duplicate symbols for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [lib/libstorm-parsers.dylib] Error 1
make[1]: *** [src/storm-parsers/CMakeFiles/storm-parsers.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 92%] Built target storm-pars
make: *** [all] Error 2
```

Could you please advise me what should I do?
Thank you for your help.

**Comment by tquatmann** [comment]
This appears to be related to [an already fixed issue](https://github.com/moves-rwth/storm/issues/320).
Could you try a `brew update` and also check that `brew info stormchecker` says 

```
==> stormchecker: stable 1.8.0, HEAD
```

**Comment by alebugariu** [comment]
Yes, I've tried:
`brew update`
Already up-to-date.
`brew info stormchecker`
`==> moves-rwth/storm/stormchecker: stable 1.8.0, HEAD`

**Comment by volkm** [comment]
The issue is indeed related to https://github.com/moves-rwth/storm/issues/320. The fix only applies the workaround for Boost 1.81. The latest version Boost 1.82 however still contains this issue and I could reproduce it on my machine. We will publish a fix soon.

**Comment by volkm** [comment]
The latest Storm release 1.8.1 contains the necessary fix for Boost. Installing via homebrew should work again.
